static/scripts/dashboard/dashboardCtrl.js

```diff
@@ -787,6 +834,16 @@
           if (err)
             return next(err);
 
+          if (bag.name === 'auth') {
+            // remove 'Keys' off the masterName to
+            // get the key name for the installForm
+            var key = bag.systemIntegration.masterName.substr(0,
```

Comment:
- we cannot do this. systemIntegrations are not being translated here correctly because of which such logic is being added. the `name` field in `systemIntegrations` is **not** the primary key and the logic in this file assumed it was. 
- the `$scope.vm.installForm` variable should create an array (or a map) for each integration name. so it should be something like
```
$scope.vm.installForm = {
"api": [{masterName: url, data: {}}]
"auth": [{masterName: githubKeys, data{}}, {masterName: bitbucketKeys, data: {}}],
}
```